### DISCUSSION TITLE
Understanding hyper63

### DISCUSSION THREAD

**Discussion by Benanna2019** [hyper63/atlas#77] [Q&A] [answered]
So, in an attempt to have a better understanding of hype63, hyper63 allows me to keep all of that business logic secure so that if I need to move it around I can essentially 'unplug' it from where it is and then 'plug in' somewhere else?

Clarification Question: So, maybe this is a misunderstanding, but hyper63 helps take care of the general stuff, like database connections and the 'necessities' so that the developer team can focus on making the product unique/special/awesome?

**Comment by twilson63** [comment, marked as answer]
Yes, hyper63 gives you common services that any application needs. This way the team is free to focus on the special logic and algorithms that make your product valuable. This effect dramatically reduces technical debt, instead of having to manage services, your team focuses on the application rules.

41% of a developers time is spent on maintenance, bugs and technical debt. hyper63 can help you cut that in half.
